Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -312,7 +312,17 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
         }
     }()
 
-    public lazy var tunnelFailureMonitor = NetworkProtectionTunnelFailureMonitor(tunnelProvider: self)
+    lazy var deviceManager: NetworkProtectionDeviceManagement = NetworkProtectionDeviceManager(
+        settings: self.settings,
+        tokenStore: self.tokenStore,
+        keyStore: self.keyStore,
+        errorEvents: self.debugEvents,
+        isSubscriptionEnabled: self.isSubscriptionEnabled
+    )
+
+    private lazy var tunnelFailureMonitor = NetworkProtectionTunnelFailureMonitor(handshakeReporter: adapter)
+    private lazy var serverFailureMonitor = NetworkProtectionTunnelFailureMonitor(handshakeReporter: adapter)
```

Comment:
Why do we need two failure monitors?

Reply:
God that’s embarrassing actually… it made more sense when I was configuring them differently, but now I’m not. So really just need the one.